launch.go

```diff
@@ -403,12 +403,14 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 
 	apiURL, _ := api.GetAPIURL()
 
+	isCI := strconv.FormatBool(!api.IsLocal())
```

Comment:
IsLocal() method is used only here?
If so, is it better pass isLocal as an argument?
Then the API interface does not need to have IsLocal function.

Reply:
I didn't want to add more arguments because there were too many arguments for `launch`

Reply:
I feel strange the API have IsLocal function. 
I think the good point to use Interface is you do not need to specify the interface is.

Reply:
If the arguments are too many. Maybe we can use struct for the arguments.

Reply:
Remove `API.IsLocal()` and add argument to `launch` and `launchAction`